src/test/java/org/apache/sling/models/impl/via/ChildResourceViaProviderTest.java

```diff
@@ -0,0 +1,63 @@
+package org.apache.sling.models.impl.via;
```

Comment:
Please move to below license to match other files.

Reply:
done